### PR TITLE
[WIP] views/utils: Fetch last 30 non-muted messages.

### PR DESCRIPTION
**Pull request by sumanthvrao**
Muted messages are not displayed in default All_message view. This causes less then 30 messages to be displayed in case there were any previous muted messages. We now keep on fetching until there are 30 non-muted messages, which can be displayed in default view.

(This is a step in the direction of solving #326)

### CODE REVIEW

**Comment by sumanthvrao** [comment]
@neiljp Let me know your thoughts on this. Whether this is the right direction to proceed in.

File: zulipterminal/ui_tools/utils.py [hunk 1]
```diff
@@ -45,6 +45,9 @@ def create_msg_box_list(model: Any, messages: Union[None, Iterable[Any]]=None,
         last_msg = msg
     if focus_msg is not None:
         model.set_focus_in_current_narrow(focus_msg)
+    model.muted_msg_count = muted_msgs
+    if message_list:
+        model.last_msg_id = message_list[0]['id']
```
Comment (neiljp) [inline zulipterminal/ui_tools/utils.py]: I'm not sure if you wanted a review of this approach, but setting these attributes is only applicable to each fragment of message list created using this function, which seems strange.

**Comment by sumanthvrao** [comment]
[Last 30 messages crash](https://trello.com/c/tjT8F5uo/4-last-30-messages-crash)

**Comment by sumanthvrao** [comment]
This is being developed in another PR #446 with a better approach. I'm closing this due to the same reason.

**Pull request closed without merging**